activesupport/lib/active_support/testing/parallelize_when_suitable_executor.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  module Testing
+    class ParallelizeWhenSuitableExecutor
```

Comment:
This should be no-doc'd

```suggestion
    class ParallelizeExecutor # :nodoc:
```